### PR TITLE
Adding Transfer manager

### PR DESCRIPTION
**Pull request by leonprou**

### CODE REVIEW

**Comment by leonprou** [comment]
The transfer manager implemented in the fuse-studio-contracts repo: https://github.com/ColuLocalNetwork/fuse-studio-contracts/pull/1

File: test/poa20_test.js [hunk 1]
```diff
@@ -81,6 +87,52 @@ async function testERC677BridgeToken(accounts, rewardable) {
     })
   })
 
+  describe('#tranferManager', async() => {
```
Comment (LiorRabin) [inline test/poa20_test.js]: missing "f" --> transferManager :)

File: test/poa20_test.js [hunk 2]
```diff
@@ -81,6 +87,52 @@ async function testERC677BridgeToken(accounts, rewardable) {
...
+      (await token.transferManager()).should.be.equal(ZERO_ADDRESS);
+
+      await token.setTransferManager(invalidContractAddress).should.be.rejectedWith(ERROR_MSG);
+      (await token.transferManager()).should.be.equal(ZERO_ADDRESS);
+
+      await token.setTransferManager(ZERO_ADDRESS).should.be.rejectedWith(ERROR_MSG);
+      (await token.transferManager()).should.be.equal(ZERO_ADDRESS);
+    })
+
+    it('can set transfer manager contract', async () => {
```
Comment (LiorRabin) [inline test/poa20_test.js:127]: duplicate test

**Review by LiorRabin — APPROVED** [review]
cool, just two mini-minor things

File: test/poa20_test.js (hunk 1 above)
Comment (bejavu) [reply test/poa20_test.js]: s*

**Review by bejavu — APPROVED** [review]

**Pull request merged**